frontend/package.json

```diff
@@ -4,6 +4,9 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.5.10",
+    "@graphql-codegen/typed-document-node": "^2.2.8",
+    "@graphql-codegen/typescript": "^2.4.8",
+    "@graphql-codegen/typescript-operations": "^2.3.5",
```

Comment:
can we save these as dev dependencies? unless we are going to use a build pipeline that will require these. 
yarn remove etc...
yarn add etc... -d

Reply:
its possible we won't be committing the generated code (will regenerate on each deployment), in which case we'll need the generator in production

Reply:
Fair point, would this be the same in the back end as well? Currently the backend has the code gen cli in the dev dependencies

Reply:
good idea, but want to do in different pr because I'm lazy as hell

Reply:
You say lazy, its just good separation. No scope creep on this PR